### PR TITLE
Adding a more complete demo to Dualsphysics

### PR DESCRIPTION
**Pull request by sarmento**
We were missing info for DualSPHysics:

https://docs.staging.inductiva.ai/simulators/DualSPHysics.html

### CODE REVIEW

File: docs/simulators/DualSPHysics.md [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
+
+Thereafter, there are other commands that allow you to post-process the results.
+Examples of this are:
 - `partvtk`: to generate VTK files with the particle trajectories;
 - `isosurface`: to generate VTK files with the isosurfaces of the fluid;
 - `measuretool`: to generate CSV files with measurements of the fluid properties.
 
-For an extensive list of commands please check the DualSPHysics [documentation](https://dual.sphysics.org/). 
-For the API, you can passed them in lower case and we will handle the rest for you!
+For an extensive list of commands please check the DualSPHysics
```
Comment (maiyahershey) [inline docs/simulators/DualSPHysics.md:21]: For an extensive list of commands, please refer to the DualSPHysics documentation. You can pass the API commands in lowercase, and we will handle the rest for you!

File: docs/simulators/DualSPHysics.md [hunk 2]
```diff
@@ -48,6 +52,395 @@ task.download_outputs()
 
 ```
 
+Below, are going to show how to run a slighly more complex examples that come
```
Comment (maiyahershey) [inline docs/simulators/DualSPHysics.md:55]: Below, we will demonstrate how to run a slightly more complex example included in the DualSPHysics distribution, located in the examples folder.

File: docs/simulators/DualSPHysics.md [hunk 3]
```diff
@@ -48,6 +52,395 @@ task.download_outputs()
...
+
+# Running `examples/chrono/09_Turbine`
+We are going to run the simulation stored in `examples/chrono/09_Turbine`.
+
+![Video produced with Paraview using VTK files generated by DualSPHysics](turbine30fps.gif)
+
+Originally, this simulation is orchestrated by a shell script that receives
+parameters from the user. The script takes those parameters, performs some 
+initial configurations and then runs a series of Dualphysics commands, including
+pre-processing, simulation and a number of post-processing steps.
```
Comment (maiyahershey) [inline docs/simulators/DualSPHysics.md:69]: including pre-processing, simulation, and a number of post-processing steps.

File: docs/simulators/DualSPHysics.md [hunk 4]
```diff
@@ -48,6 +52,395 @@ task.download_outputs()
...
+```bash
+# CODES are executed according the selected parameters of execution in this testcase
+${gencase} ${name}_Def ${dirout}/${name} -save:all
+if [ $? -ne 0 ] ; then fail; fi
+
+${dualsphysicscpu} ${dirout}/${name} ${dirout} -dirdataout data -svres
+if [ $? -ne 0 ] ; then fail; fi
+```
+
+The post-processing commands follow, with calls to `partvtk`, `partvtkout`,
```
Comment (maiyahershey) [inline docs/simulators/DualSPHysics.md:174]: The post-processing commands include calls to `partvtk`, `partvtkout`, `boundaryvtk`, and `isosurface`. These commands generate the data required to produce the visualization displayed above using ParaView.”

File: docs/simulators/DualSPHysics.md [hunk 5]
```diff
@@ -48,6 +52,395 @@ task.download_outputs()
...
+`boundaryvtk` and `isosurface`. These are the commands that generate the data
+required for producing the visualization shown above with ParaView.
+
+There are also a number of variables being set in the beginning of the script.
```
Comment (maiyahershey) [inline docs/simulators/DualSPHysics.md:178]: The script also sets several variables at the beginning, some of which relate to the paths for DualSPHysics commands, assuming a local installation. In our Inductiva Python script, we won’t need these, as the API automatically handles all command paths on the remote machine.

File: docs/simulators/DualSPHysics.md [hunk 6]
```diff
@@ -48,6 +52,395 @@ task.download_outputs()
...
+remote machine. There are also a number of variable related to filenames and 
+input/output directories:
+```bash
+export name=CaseTurbine
+export dirout=${name}_out
+export diroutdata=${dirout}/data
+```
+For a matter of convenience, we will want to keep the control over input / 
+output directories, so we will convert and keep these variables in our python
+script. We will also take the chance to explicitly add a few more output
```
Comment (maiyahershey) [inline docs/simulators/DualSPHysics.md:191]: Additionally, we’ll take this opportunity to explicitly add a few more output directories and make the process more Pythonic. So, our Python script should look something like this:

File: docs/simulators/DualSPHysics.md [hunk 7]
```diff
@@ -48,6 +52,395 @@ task.download_outputs()
...
+# make them more pythonic and we will add a few others for readibility
+name="CaseTurbine"
+dirout=f"{name}_out"
+data_dirout=f"{dirout}/data"
+particles_dirout=f"{dirout}/particles"
+boundary_dirout=f"{dirout}/boundary"
+surface_dirout=f"{dirout}/surface"
+```
+
+Now, we need to convert the 7 command lines to strings, while making use of
```
Comment (maiyahershey) [inline docs/simulators/DualSPHysics.md:204]: Next, we need to convert the seven command lines into strings, while making use of the variables we just defined. We’ll also be able to call the DualSPHysics commands (such as gencase, dualsphysics, partvtk, etc.) directly, as they are all pre-installed on the machine we will spin up later.

File: docs/simulators/DualSPHysics.md [hunk 8]
```diff
@@ -48,6 +52,395 @@ task.download_outputs()
...
+    f"dualsphysics {dirout}/{name} {dirout} -dirdataout data -svres",
+    f"partvtk -dirin {data_dirout} -savevtk {particles_dirout}/PartFloating -onlytype:-all,+floating",
+    f"partvtk -dirin {data_dirout} -savevtk {particles_dirout}/PartFluid -onlytype:-all,+fluid",
+    f"partvtkout -dirin {data_dirout} -savevtk {particles_dirout}/PartFluidOut -SaveResume {particles_dirout}/_ResumeFluidOut",
+    f"boundaryvtk -loadvtk AutoActual -motiondata {data_dirout} -savevtkdata {boundary_dirout}/Turbine -onlytype:floating -savevtkdata {boundary_dirout}/Bound.vtk -onlytype:fixed",
+    f"isosurface -dirin {data_dirout} -saveiso {surface_dirout}/Surface"
+]
+
+```
+The rest of the python script follows the usual patter. Here is the finaly
```
Comment (maiyahershey) [inline docs/simulators/DualSPHysics.md:221]: The rest of the Python script follows the usual pattern. Here is the final resulting script, ready to be executed:

File: docs/simulators/DualSPHysics.md [hunk 9]
```diff
@@ -48,6 +52,395 @@ task.download_outputs()
...
+task.wait()
+
+# Shut down the machine.
+machine_group.terminate()
+
+# Just see some stats for now. We will download the results later.
+task.print_summary()
+```
+
+Now we can run this script, and the result should be something like:
```
Comment (maiyahershey) [inline docs/simulators/DualSPHysics.md:282]: Now, we can run this script, and the output should look something like this:

File: docs/simulators/DualSPHysics.md [hunk 10]
```diff
@@ -48,6 +52,395 @@ task.download_outputs()
...
+```
+Downloading and decompressing data will take a few minutes (depending on your
+network connection):
+```bash
+Downloading simulation outputs to inductiva_output/u8v7p1v7wfyvvkyc0iq0s632k/output.zip...
+100%|█████████████████████████████████████████████████████████████████████████████| 3.52G/3.52G [04:43<00:00, 12.4MB/s]
+Uncompressing the outputs to u8v7p1v7wfyvvkyc0iq0s632k...
+```
+
+As usual, the results are placed inside the `inductiva_output` folder, and then
```
Comment (maiyahershey) [inline docs/simulators/DualSPHysics.md:409]: As usual, the results are placed in the inductiva_output folder, within a subfolder named after the task. Earlier, we set a variable for the internal directory where all outputs would be placed (`dirout`), which was instantiated as CaseTurbine_out. Let’s check its contents:

**Review by maiyahershey — CHANGES_REQUESTED** [review]
Just a few typo mistakes, otherwise it is good to go!

File: docs/simulators/DualSPHysics.md (hunk 1 above)
Comment (sarmento) [reply docs/simulators/DualSPHysics.md:21]: Done.

File: docs/simulators/DualSPHysics.md (hunk 2 above)
Comment (sarmento) [reply docs/simulators/DualSPHysics.md:55]: Done.

File: docs/simulators/DualSPHysics.md (hunk 3 above)
Comment (sarmento) [reply docs/simulators/DualSPHysics.md:69]: Done.

File: docs/simulators/DualSPHysics.md (hunk 4 above)
Comment (sarmento) [reply docs/simulators/DualSPHysics.md:174]: Done.

File: docs/simulators/DualSPHysics.md (hunk 5 above)
Comment (sarmento) [reply docs/simulators/DualSPHysics.md:178]: Done!

File: docs/simulators/DualSPHysics.md (hunk 6 above)
Comment (sarmento) [reply docs/simulators/DualSPHysics.md:191]: Done!

File: docs/simulators/DualSPHysics.md (hunk 7 above)
Comment (sarmento) [reply docs/simulators/DualSPHysics.md:204]: Done!

File: docs/simulators/DualSPHysics.md (hunk 8 above)
Comment (sarmento) [reply docs/simulators/DualSPHysics.md:221]: Done!

**Pull request merged**